flink-table/flink-table-runtime-blink/src/test/java/org/apache/flink/table/runtime/functions/SqlDateTimeUtilsTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.	See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.	You may obtain a copy of the License at
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.runtime.functions;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.lang.reflect.Method;
+import java.util.TimeZone;
+
+import static org.apache.zookeeper.server.ServerCnxn.me;
+
+/**
+ * Test cases for class {@link SqlDateTimeUtils}.
+ */
+public class SqlDateTimeUtilsTest {
```

Comment:
Can you add an Expression test to cover it instead of introducing `SqlDateTimeUtilsTest `?

Reply:
Sure. Thanks 

> > Fix a bug in SqlDateTimeUtils#parseToTimeMillis: If parameter "1999-12-31 12:34:56.123" is used, it should return 123. But it returns 230 now.
> 
> It is 230 not 1230?

It should be 1230. Thanks for your careful review.

Reply:
Thanks for your suggestion. I think it is generally reasonable. However, Expression test is in flink-table-common module, which has no access to class SqlDateTimeUtils.
Can you please suggest another place to add this test?